orchestration/dags/jobs/dbt/dbt_dag_run.py

```diff
@@ -61,18 +61,18 @@
     dag=dag,
 )
 
-dbt_compile_op = BashOperator(
-    task_id="dbt_compile",
-    bash_command="dbt compile --target {{ params.target }} --select data_gcp_dbt "
+dbt_manifest_op = BashOperator(
+    task_id="dbt_make_manifest",
+    bash_command="dbt ls --target {{ params.target }} "
     + f"--target-path {PATH_TO_DBT_TARGET}",
```

Comment:
on avait cette option `--select data_gcp_dbt` pour construire le manifest uniquement sur le repo et ne pas tenir compte des packages re et elementary. Pas d'équivalent avec la commande ls ?

Reply:
le ls parse l'integralité du projet et compile le manifest les option de la commande servent an changer l'output: 
"Today[Jun 1, 2021], the fastest way to write a manifest is to use dbt ls (with --exclude fqn:* source:* exposure:* to avoid printing a bunch of noise to stdout)"
https://github.com/dbt-labs/dbt-core/issues/3410